tests/Export/ProductIdExportTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Tests\Export;
+
+use FINDOLOGIC\FinSearch\Export\ProductIdExport;
+use FINDOLOGIC\FinSearch\Export\XmlExport;
+use Shopware\Storefront\Framework\Routing\Router;
+
+class ProductIdExportTest extends XmlExportTest
```

Comment:
It seems like `$exporter = $this->getExport();` is used for every test, so class property and `setup` method?

Reply:
Basically, doing the stuff from `getExport` in `setup` and set `$this->exporter`

Reply:
I tried doing that, but I can't, since the `crossSellingCategories` can only be added once to the constructor. Therefore I'd have to manipulate the cross selling categories first, before creating a new instance. See test `testProductsInCrossSellCategoriesAreNotWrappedAndErrorIsLogged`.

Reply:
tbd